eis_toolkit/cli.py

```diff
@@ -748,7 +755,7 @@ def compute_pca_raster_cli(
 def compute_pca_vector_cli(
     input_vector: INPUT_FILE_OPTION,
```

Comment:
Can INPUT_FILES_ARGUMENT be used here as well? Just for consistency.
```suggestion
    input_vector: INPUT_FILES_ARGUMENT,
```

Reply:
`INPUT_FILES_ARGUMENT` is used only when multiple datasets (rasters) are needed